src/coreclr/inc/formattype.h

```diff
@@ -6,6 +6,10 @@
 
 #include "corpriv.h"					// for IMDInternalImport
 
+#ifdef _DEBUG
+#include "../md/debug_metadata.h"   // for Debug_ReportError
```

Comment:
Where does the code use `Debug_ReportError` ? I do not see it in this file. Can this include be closer to where it is used?